subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -1674,32 +1686,17 @@ static int lwm2m_exec_handler(struct lwm2m_engine_obj *obj,
 static int lwm2m_delete_handler(struct lwm2m_engine_obj *obj,
 				struct lwm2m_engine_context *context)
 {
-	struct lwm2m_obj_path *path = context->path;
-	struct lwm2m_engine_obj_inst *obj_inst;
-
-	SYS_LOG_DBG(">> DELETE [path:%u/%u/%u(%u)]",
-		path->obj_id, path->obj_inst_id, path->res_id, path->level);
+	struct lwm2m_obj_path *path;
 
-	if (!obj || !context) {
+	if (!context) {
 		return -EINVAL;
 	}
 
-	obj_inst = get_engine_obj_inst(path->obj_id, path->obj_inst_id);
-	if (!obj_inst) {
-		return -ENOENT;
-	}
-
-	engine_unregister_obj_inst(obj_inst);
-	obj->instance_count--;
-#ifdef CONFIG_LWM2M_RD_CLIENT_SUPPORT
-	engine_trigger_update();
-#endif
-
-	if (obj->delete_cb) {
-		return obj->delete_cb(path->obj_inst_id);
-	}
+	path = context->path;
+	SYS_LOG_DBG(">> DELETE [path:%u/%u/%u(%u)]",
```

Comment:
I left this in the original code by accident.  Can you remove it here?

Reply:
sure